File: iosApp/iosApp/state/MainState.swift
//
//  MainState.swift
//  iosApp
//
//  Created by Roberto García Romero on 5/2/24.
//  Copyright © 2024 orgName. All rights reserved.
//

import Foundation
import shared

struct MainStateSwift {
    var isLoading: Bool = false
    var tapas: [TapaItemBo] = []
    var filteredTapas: [TapaItemBo] = []
    var logout: Bool = false
    var provinces: [String] = []
    var error: String? = nil
}

File: shared/src/commonMain/kotlin/com/rgr/rutappa/app/state/MainState.kt
package com.rgr.rutappa.app.state

import com.rgr.rutappa.domain.error.RemoteConfigError
import com.rgr.rutappa.domain.model.TapaItemBo

data class MainState(
    val isLoading: Boolean = false,
    val tapas: List<TapaItemBo> = emptyList(),
    val filteredTapas: List<TapaItemBo> = emptyList(),
    val logout: Boolean = false,
    val provinces: List<String> = emptyList(),
    val error: RemoteConfigError? = null
)
